src/cli/run.ts

```diff
@@ -135,6 +135,7 @@ export const runTask = async (
     outputTargets: config.outputTargets ?? [],
     sys: sys ?? config.sys ?? coreCompiler.createSystem({ logger }),
     testing: config.testing ?? {},
+    rootDir: config.rootDir ?? '/',
```

Comment:
I haven't tested it out when calling from the API...I thought it would be a safe default based on the fact that the validation function was doing that already. 

I think that the node `path` module on windows will resolve correctly a posix-style path if it's handed one, I don't have a windows machine handy to test on but if I do the following:

```ts
require("path").win32.resolve("/")
```

I get `"\\"` which is a valid win32 path which I think should work? I wish all computers just had one standard for things like this!